src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Card/VerificationDto.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    internal class VerificationDto
+    {
+        public bool Direct { get; set; }
+        public string CardBrand { get; set; }
+        public string CardType { get; set; }
+        public string IssuingBank { get; set; }
+        public string RecurrenceToken { get; set; }
+        public string MaskedPan { get; set; }
+        public string ExpiryDate { get; set; }
+        public string PanToken { get; set; }
+        public bool PanEnrolled { get; set; }
+        public string IssuerAuthorizationApprovalCode { get; set; }
+        public string AcquirerTransactionType { get; set; }
+        public string AcquirerStan { get; set; } 
+        public string AcquirerTerminalId { get; set; }
+        public DateTime AcquirerTransactionTime { get; set; }
```

Comment:
This isn't how it's done on other places though. Other places do have DateTime in DTO objects as well?

Reply:
Okay, I think that may be a mistake. It gives us much more flexibility, control, and improved error messages to do the conversion in the mapping instead of relying on the serializer to do so.